packages/from-to-location-picker/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@opentripplanner/from-to-location-picker",
+  "version": "0.0.1",
+  "description": "Location Picker Component",
+  "keywords": [
+    "sample",
+    "component"
+  ],
+  "author": "c-harper <charper@mapyours.com>",
+  "homepage": "https://github.com/TriMetPDX/otp-ui#readme",
+  "license": "MIT",
+  "main": "lib/sample.js",
+  "module": "src/sample.js",
+  "peerDependencies": {
+    "icons": "^0.0.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "styled-components": "^4.4.1"
+  },
+  "publishConfig": {
+    "registry": "https://registry.yarnpkg.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TriMetPDX/otp-ui.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
+  "bugs": {
+    "url": "https://github.com/TriMetPDX/otp-ui/issues"
+  },
+  "dependencies": {
+    "@opentripplanner/location-icon": "0.0.1",
+    "@storybook/addon-actions": "^5.2.8"
```

Comment:
Please remove, this is available in the root package.json

Reply:
Will change for now.